evil-smartparens.el

```diff
@@ -75,14 +75,12 @@ list of (fn args) to pass to `apply''"
 (defun evil-sp--get-endpoint-for-sp-kill-sexp ()
   (unwind-protect
       (progn
-        (push major-mode sp-navigate-consider-stringlike-sexp)
         (evil-sp--new-ending (point)
```

Comment:
I think the intention here is to have sp consider whatever mode we're in now to be a lisp mode temporarily.  Is that no longer necessary?

Reply:
I cannot find where I found it is no longer necessary, but I am sure that I saw something about it being the default behavior, which is why `sp-navigate-consider-stringlike-sexp` no longer exists.

Reply:
Although, I am now getting a `search failed. This means there is unmatched expression somewhere or we are at the beginning/end of the file` when deleting to the end of a line with `D`. It still does what I want it to do, though, which it id not before deleting references to `sp-navigate-consider-stringlike-sexp`.